src/functions/underline.js

```diff
@@ -24,17 +24,19 @@ defineFunction({
         // Build the inner group.
         const innerGroup = html.buildGroup(group.value.body, options);
 
-        // Create the line above the body
+        // Create the line to go below the body
         const line = buildCommon.makeLineSpan("underline-line", options);
 
         // Generate the vlist, with the appropriate kerns
         const vlist = buildCommon.makeVList({
             positionType: "top",
             positionData: innerGroup.height,
             children: [
-                {type: "kern", size: line.height},
+                // The SVG image is 5x as tall as the line.
+                // The bottom 2/5 of the image is blank and acts like a kern.
+                // So we omit the kern that would otherwise go at the bottom.
                 {type: "elem", elem: line},
-                {type: "kern", size: 3 * line.height},
+                {type: "kern", size: 5 * line.height},
```

Comment:
Are these `elem` and `kern` children overlapping somehow?

Reply:
Yes. 
![underline](https://user-images.githubusercontent.com/16403058/34139800-989f4192-e42b-11e7-8d20-93bf26d3ae6a.png)